test/e2e/common/node/init_container.go

```diff
@@ -235,7 +235,9 @@ var _ = SIGDescribe("InitContainer [NodeConformance]", func() {
 
 		framework.ExpectEqual(len(endPod.Status.InitContainerStatuses), 2)
 		for _, status := range endPod.Status.InitContainerStatuses {
-			framework.ExpectEqual(status.Ready, true)
+			if !status.Ready {
+				framework.Fail("init container should be in Ready status")
```

Comment:
Can you use Failf here and add the name of the init container here? That seems like useful information to have in case of a failure.

Reply:
sure, fixing